src/World.h

```diff
@@ -79,7 +79,7 @@ typedef cItemCallback<cFlowerPotEntity>    cFlowerPotCallback;
 
 
 
-
+const int CHUNK_CHANCE_ACCURACY = 100000;
```

Comment:
You wouldn't even have to worry about doing programmatic gymnastics to avoid saving unused chunks, or sorta try and wait for other threads to do work.